auth/auth.go

```diff
@@ -62,6 +64,17 @@ func NewClient(ctx context.Context, conf *internal.AuthConfig) (*Client, error)
 		err    error
 	)
 
+	baseURL := defaultAuthURL
+	if authEmulatorHost := os.Getenv("FIREBASE_AUTH_EMULATOR_HOST"); authEmulatorHost != "" {
```

Comment:
Should probably define a local constant for the env variable name.

Reply:
I'd define a constant. https://github.com/firebase/firebase-admin-go/pull/414/commits/11d9970757fb1ee5e6ff0992ed219d8bfd4cbdc0#diff-f5c9a39fac1719f2a73593994b931d8a874449548e8a3753e1be46e761e503eaR33